packages/check/tests/cosmwasm_check_tests.rs

```diff
@@ -0,0 +1,59 @@
+use assert_cmd::prelude::*;
+use predicates::prelude::*;
+use std::process::Command;
+
+#[test]
```

Comment:
Nice to have some tests for cosmwasm-check now!
Would you mind adding another one showing that `floaty.wasm` is rejected because of floating point operations?

Reply:
@chipshort 
Thanks, will try.

However, it appears that using clap version 4 with rustic 1.60.0 is not possible.
I attempted to use the lowest possible version of clap from the 4.x series, which is 4.0.0, but it requires rustc 1.64.0

Reply:
Nice tests. We can leave this PR around for when the CI moves the MSRV rust version to 1.64.0 then.

Reply:
Feel free to bump the min Rust version from 1.60.0 to 1.64.0 in CI (in a separate PR). This is 8 months old and wasmvm works fine with more recent versions than that already.

Reply:
https://github.com/CosmWasm/cosmwasm/pull/1680

Reply:
Have added a test for floaty.wasm file.